### PR TITLE
Style button, sign up form, and `top-four` article section

### PR DESCRIPTION
**Pull request by slocker3**
- styles for `inputs`
- layout for `top-four` section
- *note* MailChimp code for signup

### CODE REVIEW

**Comment by slocker3** [comment]
@jeremyfelt #reviewmerge

**Comment by slocker3** [comment]
Do we need a shortcode for the MailChimp embed?

File: style-guide/home.html [hunk 1]
```diff
@@ -110,6 +110,70 @@ <h2 class="sub-call">Tell us how we&#8217;re doing</h2>
...
+							</div>
+						</div>
+					</section>
+
+					<section class="row single gutter pad-top bg-none">
+						<div class="column one" style="">
+							<!-- Begin MailChimp Signup Form -->
+<link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
+<style type="text/css">
+	#mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; width:100%;}
```
Comment (jeremyfelt) [inline style-guide/home.html]: Do we like this CSS or can we just remove it?

**Comment by jeremyfelt** [comment]
Yeah, I can create the shortcode for implementing this in the next PR.

**Comment by slocker3** [comment]
Styles stripped

**Comment by jeremyfelt** [comment]
* Added the `insider_subscribe_form` shortcode.
* Adjusted the "Subscribe" button height to match that of the input field - 40px.

@slocker3 The `top-four` section needs work on mobile before this can ship. Pull down changes on this branch before adding to it :)

screen shot 2017-10-02 at 4 12 55 pm

**Pull request merged**